visualizer/index.html

```diff
@@ -156,70 +163,6 @@
       -webkit-user-select: none;
     }
 
-    #exampleList {
-      list-style: none;
-      border-top: 1px solid #ddd;
-      margin: 0;
-      padding: 0;
-    }
-
-    #exampleList .example {
-      border-bottom: 1px solid #eee;
-      color: #999;
-      display: flex;
-      margin: 0;
-      white-space: nowrap;
-    }
-
-    #exampleList .example code {
-      border-left: 4px solid #bbb;
-      cursor: default;
-      flex: 1;
-      overflow: hidden;
-      padding: 6px 4px 6px 8px;
-      text-overflow: ellipsis;
-
-      /* Weirdly, it won't shrink smaller than its content unless we give it an explicit width. */
-      width: 1px;
-    }
-
-    #exampleList .example.fail code {
-      border-left: 4px solid #d44950;
-    }
-
-    #exampleList .example.pass code {
-      border-left: 4px solid #58B958;
-    }
-
-    #exampleList .example.selected {
-      color: #333;
-      background-color: #f7f7f7;
-    }
-
-    #exampleList code {
-      font-family: Menlo, Monaco, sans-serif;
-      font-size: 12px;
-    }
-
-    #exampleList code:empty::before {
-      content: '\a0';  /* Insert &nbsp; to keep the correct height. */
```

Comment:
Did you mean to delete this rule? I think we need it, to ensure that the examples keep the same height even when there is no text in them.